cms/jinja2/templates/pages/information_page.html

```diff
@@ -0,0 +1,27 @@
+{% extends "templates/base_page.html" %}
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+
+{% block main %}
+    <p>{{ page.summary }}</p>
+
+    {% if page.last_updated %}
+        <h5>Last Updated: {{ page.last_updated }}</h5>
+    {% endif %}
+
+    {% include_block page.content %}
+
+    {% if related_pages %}
```

Comment:
To make use of this you need to add this to the content panel:

`InlinePanel("page_related_pages", label="Related pages")`